core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceInputCheckerServiceSpec.js

```diff
@@ -26,11 +26,11 @@
      $injector.get('MultipleChoiceInputCheckerService');
    }));
 
-   it('Should return true if interaction data is compatible', function(){
-     expect(multipleChoiceInputCheckerService.isValid()).toEqual(true);
+   it('Should return true if interaction data is compatible', function() {
+     expect(multipleChoiceInputCheckerService.isValid()).toBe(true);
```

Comment:
pass args to `multipleChoiceInputCheckerService.isValid()` as it expects two arguments.